src/pixi-multistyle-text.ts

```diff
@@ -1,3 +1,25 @@
+import {
+  Text,
+  TextMetrics,
+  TextStyle as PixiTextStyle,
+  TEXT_GRADIENT,
+  Texture,
+  interaction,
+  utils,
+  Rectangle,
+} from "pixi.js";
+
+const PIXI = {
```

Comment:
Instead of creating a `PIXI` object, I'd just update all references in the file to drop `PIXI.`. So instead of doing `PIXI.Text`, just use `Text` from the import.

Also, since we have the import, we can also remove this comment I believe.
```
<reference types="pixi.js" />
```

Reply:
I've made these changes. I renamed, for example, utils as pixiUtils to be less ambiguous.